src/components/Card/docs/index.story.tsx

```diff
@@ -100,6 +103,50 @@ export default {
         { title: 'API', sections: [api()] },
         { title: 'TestKit', sections: [testkit()] },
         { title: 'Playground', sections: [playground()] },
+        {
+          title: 'Settings Panel',
+          sections: [
+            settingsPanel({
+              example: <CardExtendedExample />,
+              rawSource: ExtendedRawSource,
+              rawCSSSource: ExtendedCSSRawSource,
+              params: {
+                numbers: [
+                  {
+                    label: 'Border Width',
+                    wixParam: 'borderWidth',
+                    defaultNumber: 1,
```

Comment:
```suggestion
                    default: 1,
```
it can be implied that it's number, cause it's part of numbers array. same for colors

unless, maybe it's too much work to change now?

Reply:
This is actually the APi of the `BaseUI` component that we import from `wix-base-ui`